src/managers/AdManager/AdManager.ts

```diff
@@ -21,82 +26,91 @@ class AdManager {
 
   private page: TPages | null;
   private productsIds: ReturnType<typeof getProductsIds> | [];
-  private websiteId: number;
 
   public injectAdnPixelScript = () => {
     const area = this.page ? this.mapPageToArea(this.page) : null;
 
     if (window.dlApi && dlApi.fetchNativeAd) {
-      dlApi.area = area || '';
-      dlApi.keyvalues = {
+      dlApi.addKeyValue('area', area || '');
```

Comment:
tutaj użycie `dlApi.area` było ok.
Metoda `dlApi.addKeyValue` służy konkretnie do ustawiania danych w `dlApi.keyvalues`, a nie do wszystkiego w `dlApi`
więc przykładowo można tego użyć tak:
```js
console.log('foo:', dlApi.keyvalues['foo']) // "foo: undefined"
dlApi.addKeyValue('foo', 'bar')
console.log('foo:', dlApi.keyvalues['foo']) // "foo: bar"
```

Reply:
Tak jak napisał @ravkr tutaj należy poprawić tę linijkę na
```suggestion
      dlApi.area = area || 'ros'
```

`ros` powinien być domyślną wartością w przypadku gdy nie uda się zmapować wartości na `area`